requirements.txt

```diff
@@ -1,3 +1,3 @@
 --extra-index-url https://europe-west3-python.pkg.dev/bionic-unity-294411/helicon-python-api/simple/
 
-Helicon-Python-Client==1.0.0.dev3
\ No newline at end of file
+Helicon-Python-Client==1.0.0.dev5
```

Comment:
is it the version supposed to work for CLOUD environment?

Reply:
no, this is using the latest version of the library, it won't be merged before we are ready. Said that, the actual version and the latest version are both compatible with the server in cloud environment

Reply:
if you are sure about it, we can go